src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -28,6 +38,15 @@ const InputText = ({
   return (
     <div className={'form-group ' + className}>
       <label htmlFor={inputId}>{label}</label>
+      {helperText !== '' && (
+        <p className="form-group__p--margin-zero">
+          {helperText}
+          <a href={linkHref} rel="noreferrer external" target="_blank">
+            {linkText}
+            {icon === true && <FontAwesomeIcon className="helper-text__link-icon" icon={faUpRightFromSquare} />}
```

Comment:
`icon === true && ...` is equivalent to `icon && ...` or a safer `!!icon` if unsure of the type